routes/images.js

```diff
@@ -0,0 +1,16 @@
+import express from "express"
+import imageController from "../controllers/images.js";
+
+const { addImages, addImage, getImages, deleteImage } = imageController
+const router = express.Router();
+
+router.post("/", addImages);
```

Comment:
We're going to have  to change our `addImages` and `addImage` functions. we should have only one method that does this, maybe  an array, and then  we can loop through the array, if we have only one then we add only one image or if we have multiple, we add them sequentially

Reply:
I haven't done this

Reply:
done